contents/tutorials/csv-query.md

```diff
@@ -0,0 +1,131 @@
+---
+title: How to query a CSV in PostHog
+date: 2024-09-23
+author:
+ - ian-vanagas
+tags:
+ - data warehouse
+---
+
+PostHog can capture a lot of data about your users. For data it can't capture, you can leverage the [data warehouse](/data-warehouse) to add whatever data you want as a CSV.
```

Comment:
```suggestion
PostHog can capture a lot of data about your users. For data it can't capture, you can leverage the [data warehouse](/data-warehouse) to manually upload any data you'd like as a CSV
```